src/LazyOperations/Bloating.jl

```diff
@@ -0,0 +1,140 @@
+export Bloating
+
+"""
+    Bloating{N<:AbstractFloat, S<:LazySet{N}} <:LazySet{N}
+
+Type that represents a (possibly negative) uniform expansion of a convex set in
```

Comment:
here we have the word "negative" but below it is only 'positive'. as i understand it, if 0 < eps < 1 this is a contraction and otherwise for eps > 1 it is an expansion. i suggest to add this information and / or an example in the constructor's docstring.

Reply:
Yeah, sorry, this is a left-over of an old version. I only want to allow values > 1.